src/engine/thread.js

```diff
@@ -349,6 +349,7 @@ class Thread {
             if (frame.params.hasOwnProperty(paramName)) {
                 return frame.params[paramName];
             }
+            return null;
```

Comment:
Could you clarify this change some more? What was failing before this change?

Reply:
This was answered in a discussion offline:

> getParams searches for a params object in the stack. When it encounters a params object, it can check it for the parameter being searched for. After the defaults branch being merged, if its not there that implies the param is not from the current running procedure.
Since the parameter is not in the current procedure we can shortcut out of any more searching.